src/client/components/VotingBox.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const VotingBox = ({ voteForItem, votesNumber, whetherUserVoted }) => (
+  <button
+    aria-label="Vote for item"
+    className={classNames('voting-box', {
+      'voting-box__voted': whetherUserVoted
+    })}
+    type="button"
+    onClick={e => {
+      e.stopPropagation();
+      voteForItem();
```

Comment:
not sure if we should call this here directly or just invoke as a reference?  `onClick={voteForItem}`